docs/index.html

```diff
@@ -0,0 +1,291 @@
+                </span>
+              </cds-link>
+            </cds-tile>
+          </div>
+
+          <div class="card cds--col">
+            <cds-tile color-scheme="white">
+              <h2 class="project-title">Example 2</h2>
+              <div class="tags">
+                <cds-tag type="purple" title="ibm-quantum" size="sm">Algorithms</cds-tag>
+                <cds-tag type="purple" title="ibm-quantum" size="sm">Julia</cds-tag>
+              </div>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique odio, nostrum voluptatum quidem tempore dolores! Iste vero reprehenderit rerum minima! Laudantium quia nemo aliquid dolorem sit at. Aliquid, consequuntur culpa!</p>
+              <cds-link href="https://ibm.com">
+                Go to repository
+                <span class = "link" slot="icon">
+                  <svg focusable="false" preserveAspectRatio="xMidYMid meet"
+                       xmlns="http://www.w3.org/2000/svg" fill="currentColor"
+                       width="16" height="16" viewBox="0 0 16 16" aria-hidden="true"
+                       data-v-bdc8f36d="">
+                    <path d="M13,14H3c-0.6,0-1-0.4-1-1V3c0-0.6,0.4-1,1-1h5v1H3v10h10V8h1v5C14,13.6,13.6,14,13,14z"></path>
+                    <path d="M10 1L10 2 13.3 2 9 6.3 9.7 7 14 2.7 14 6 15 6 15 1z"></path>
+                  </svg>
+                </span>
+              </cds-link>
+            </cds-tile>
+          </div>
+
+          <div class="card cds--col">
+            <cds-tile color-scheme="white">
+              <h2 class="project-title">Example 3</h2>
+              <div class="tags">
+                <cds-tag type="cyan" title="ibm-quantum" size="sm">IBM Quantum</cds-tag>
+              </div>
+              <div class="description">
+                <input type="checkbox" class="read-more-input" id="extra-1"/>
+                <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique odio, nostrum voluptatum quidem tempore dolores! Iste vero reprehenderit rerum minima! Laudantium quia nemo aliquid dolorem sit at. Aliquid, consequuntur culpa! <span class="extra-text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Accusamus laudantium deleniti esse sed? Asperiores dicta minima numquam. Voluptatem molestiae itaque nesciunt omnis dolorem dicta exercitationem iste ducimus illum, asperiores maiores! Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dignissimos eos eaque expedita necessitatibus tempore repudiandae amet incidunt, quia asperiores perspiciatis, doloremque officiis corrupti ipsum! Laudantium tempora quo adipisci excepturi autem. Lorem ipsum dolor sit amet consectetur adipisicing elit. Sequi nemo placeat fugiat, error odit ullam dolorum soluta accusamus quisquam odio, eligendi libero quas perspiciatis consequuntur vero minus qui! Facere, reiciendis!</span></p>
+                <label for="extra-1" class="read-more-button"></label>
+              </div>
```

Comment:
Really good idea thinking about how tall some of the tiles could be (some descriptions are _looong_).

I think it might be better to use an [expandable tile](https://web-components.carbondesignsystem.com/?path=/story/components-tile--expandable-with-layer) within the larger tile, but I'm not 100% sure on this so appreciate other opinions.

Reply:
Agreed! This is a fantastic feature!

I'd recommend investigating the expandable tile option, if it is a simple change go for it, if it's going to require a substantial refactor then don't bother, this is fine.

Reply:
Thank you! I didn't know they existed! I have tried it, but I haven't been able to align them and make only the middle part expandable (the title and tags are above, and links are below the hidden text), but I'll continue investigating! I like how they work!

Reply:
@arnaucasau I think you should use the expandable tile within the regular tile, so something like:

```html
<tile>
  <h3>Project name</h3>
  <cds-expandable-tile>
    ...
  </cds-expandable-tile>
  Links etc.
</tile>
```